app/upgrades/v8/constants/constants.go

```diff
@@ -0,0 +1,13 @@
+// package v8constants contains constants related to the v8 upgrade.
+// It is in its own package to eliminate import cycle issues.
+package v8constants
+
+const (
+	// UpgradeName defines the on-chain upgrade name for the Osmosis v8 upgrade.
+	UpgradeName = "v8"
+
+	// UpgradeHeight defines the block height at which the Osmosis v8 upgrade is
+	// triggered.
+	// TODO: Choose upgrade height
+	UpgradeHeight = 100_000
```

Comment:
How is this going to be determined @ValarDragon @mattverse?

Reply:
We calculate the expected block height at expected time

Reply:
I see it was correctly updated to 4402000 in a recent commit - resolving